catalyst/dl/runner/gan.py

```diff
@@ -243,5 +245,85 @@ def _discriminator_train_phase(self):
             self.real_logits_output_key: real_logits
         }
 
+    def train(
+        self,
+        model: Union[Model, Dict[str, Model]],
+        loaders: "OrderedDict[str, DataLoader]",  # noqa: F821
+        callbacks: "OrderedDict[str, Callback]" = None,  # noqa: F821
+        phase2callbacks: Dict[str, List[str]] = None,
+        criterion: Criterion = None,
+        optimizer: Optimizer = None,
+        num_epochs: int = 1,
+        main_metric: str = "loss",
+        minimize_metric: bool = True,
+        checkpoint_data: Dict = None,
+        state_kwargs: Dict = None,
+        distributed_params: Dict = None,
+        monitoring_params: Dict = None,
+        logdir: str = None,
+        verbose: bool = False,
+        initial_seed: int = 42,
+        check: bool = False,
+    ) -> None:
+        """
+        :param model: models, usually generator and discriminator
```

Comment:
to google style docstring please

Reply:
Aw, I followed @asmekal style for MultiphaseRunner and GanRunner init as the examples.
I guess i have to rewrite them too.